utils/biospecimen.js

```diff
@@ -212,12 +212,11 @@ const biospecimenAPIs = async (req, res) => {
             return res.status(405).json(getResponseJSON('Only GET requests are accepted!', 405));
         }
         if(req.query.masterSpecimenId) {
-            const masterSpecimenId = req.query.masterSpecimenId;
-            if(!masterSpecimenId) return res.status(400).json(getResponseJSON('Bad request!', 400));
             const { searchSpecimen } = require('./firestore');
```

Comment:
Should we put `const { searchSpecimen } = require('./firestore');` at the top of the file?

Reply:
I think it makes more sense to leave it on line 215. You can initialize` const { searchSpecimen } = require('./firestore'); `only `if req.query.masterSpecimenId` is `true`. This saves some computation instead of initializing it at the very beginning.

Reply:
We're using conditional imports in API calls, and we can discuss its pros and cons in our use cases.